chromadb/test/property/test_filtering.py

```diff
@@ -292,3 +292,40 @@ def test_boolean_metadata(api: API) -> None:
     res = coll.get(where={"test": True})
 
     assert res["ids"] == ["1", "3"]
+
+def test_count(api: API) -> None:
```

Comment:
This test should be leveraging the generated strategies 

See how the other tests are using 

```
@given(
    collection=collection_st,
    record_set=recordset_st,
    filters=st.lists(
        strategies.filters(collection_st, recordset_st, include_all_ids=True),
        min_size=1,
    ),
)
```

That is how we fuzz the inputs and test many cases